tripal_chado/src/Plugin/TripalStorage/ChadoStorage.php

```diff
@@ -127,6 +126,31 @@ public function getNonStoredTypes() {
     return $this->getStoredTypesFilter(FALSE);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  public function isDrupalStoreByFieldNameKey(string $field_name, string $key): bool|null {
+
+    // First get our parent to do the generic check.
+    $is_required = parent::isDrupalStoreByFieldNameKey($field_name, $key);
+
+    // Then grab the property type and its storage properties in order to
+    // make any chado-specific decisions.
+    $property_type = $this->getPropertyType($field_name, $key);
+    if ($property_type === NULL) {
+      return NULL;
+    }
+    $storage_settings = $property_type->getStorageSettings();
+
+    // In chado, all table columns containing sequence are named 'residues'.
+    // We want to exclude sequences from drupal storage even if the default is TRUE.
```

Comment:
```suggestion
    // We want to exclude sequences from drupal storage even if the default is TRUE
    // because this field can contain a very large string, for example an entire chromosome.
```